source/docs/software/advanced-controls/trajectories/trajectory-generation.rst

```diff
@@ -47,13 +47,13 @@ Here is an example of generating a trajectory using clamped cubic splines for th
 
 .. tabs::
 
-   .. tab:: Java
+   .. tab:: java
 
       .. literalinclude:: examples/trajectory-generation-1/java/ExampleTrajectory.java
          :language: java
          :lines: 10-32
 
-   .. tab:: C++
+   .. tab:: cpp
```

Comment:
Why were these changed?

Reply:
I was debugging why my build earlier was failing. Technically the previous syntax was wrong, and should be using group-tabs.